docs/sources/mimir/references/http-api/index.md

```diff
@@ -566,8 +566,17 @@ GET,POST <prometheus-http-prefix>/api/v1/cardinality/label_values
 Returns realtime label values cardinality associated to request param `label_names[]` across all ingesters, for the authenticated tenant, in `JSON` format.
 It returns the series count per label value associated to request param `label_names[]`.
 
-As far as this endpoint generates cardinality report using only values from currently opened TSDBs in ingesters, two subsequent calls may return completely different results, if ingester did a block
-cutting between the calls.
+Two methods of counting are available, `inmemory` and `active`. This choice is controlled via the `count_method` parameter.
+
+The `inmemory` counting method counts the number of series using all values from currently opened TSDBs in ingesters.
+Two subsequent calls may return completely different results, if ingester did a block cutting between the calls.
```

Comment:
```suggestion
Two subsequent calls may return completely different results if an ingester cut a block between calls.
```

Is this the only case where this is true? Or is "block cutting between the calls" one example scenario (of many possibilities) which could cause this?